arlpy/uwapm.py

```diff
@@ -634,12 +876,18 @@ def run(self, env, task, debug=False, fname_base=None):
 
     def _bellhop(self, *args):
         try:
-            result = _proc.run(f'bellhop.exe {" ".join(list(args))}', 
-                        stderr=_proc.STDOUT, stdout=_proc.PIPE,
-                        shell=True)
-            if result.returncode == 127:
-                return False
-        except OSError:
+            for i in args:
+                if i.endswith(".env"):
+                    i.replace(".env", "")
+            command = "bellhop.exe {:s}".format(" ".join(list(args)))
+            # command = "\"C:/Program Files/Acoustics_Toolbox_20201102/bellhop.exe\" {:s}".format(" ".join(list(args)))
+            _proc.run(command,
+                      stderr=_proc.STDOUT, stdout=_proc.PIPE,
+                      shell=True, check=True)
+        except Exception as e:
+            raise Exception("Unable to run bellhop.exe: (1) Please check if the path of bellhop.exe has been correctly"\
+            " added to the system environment variable; "\
+                            "(2) Alternatively, manually specify the absolute path of bellhop.exe in the source code.")
```

Comment:
```suggestion
            raise Exception("Unable to run bellhop.exe: please check that bellhop.exe is on your PATH in the system environment variable")
```

Asking users to modify source code of the package is perhaps not the best option. If we wanted to allow arbitrary executable path not on the system PATH, we should provide a proper programmatic way to set it without modifying source.